src/thunderbots/software/network_input/filter/robot_filter.h

```diff
@@ -27,6 +27,7 @@ typedef struct SSLRobotData_t
 typedef struct FilteredRobotData_t
 {
     unsigned int id;
+    bool cached;
```

Comment:
What is this flag used for?

Reply:
Was just thinking that it would be nice to know if the filtered data that was received was the most recent data, or was cached for debugging purposes.

Reply:
Hmm, I'd like to keep things as minimal as possible, but lets get a second opinion here. @MathewMacDougall - your thoughts?

Reply:
We could avoid caching robot data/state entirely. My changes in #329 let the filters take the current state as a parameter to `getFiltereData` function. That way it's always clear what the initial state is, and we don't need to worry about caching at all.

Regardless, I think the `cached` flag could be a source of confusion in the future.